tests/platform_tests/sfp/test_sfpshow.py

```diff
@@ -37,7 +37,21 @@ def test_check_sfp_presence(duthosts, enum_rand_one_per_hwsku_frontend_hostname,
     parsed_presence = parse_output(sfp_presence["stdout_lines"][2:])
     for intf in dev_conn:
         if intf not in xcvr_skip_list[duthost.hostname]:
-            assert intf in parsed_presence, "Interface is not in output of '{}'".format(cmd_sfp_presence)
+            assert intf in parsed_presence, (
+                "Interface '{}' is not in the output of '{}'. "
```

Comment:
It already has failed reason, but I agree to print interface name out. Don't need to add those indications in failed reason, that's not related to the failed reason